site/src/pages/ReviewsPage/UserReviews.tsx

```diff
@@ -3,41 +3,48 @@ import SubReview from '../../component/Review/SubReview';
 import './UserReviews.scss';
 import { useAppDispatch, useAppSelector } from '../../store/hooks';
 import { selectReviews, setReviews } from '../../store/slices/reviewSlice';
+import { ReviewData } from '@peterportal/types';
+import ReviewItemGrid from '../../component/ReviewItemGrid/ReviewItemGrid';
 import trpc from '../../trpc';
 
+const UserReviewsList: FC<{ reviews: ReviewData[] }> = ({ reviews }) => {
+  return (
+    <ReviewItemGrid>
+      {reviews.length == 0 && (
+        <span>You haven't reviewed any courses yet. Look up a course you've taken to review it!</span>
+      )}
+      {reviews.map((review) => (
+        <div key={'user-review-' + review.id!}>
+          <SubReview review={review} />
+        </div>
```

Comment:
(mentioned in a previous comment) is there a reason we can't just map directly to the `SubReview`?